src/sidebar/templates/sidebar-content.html

```diff
@@ -9,7 +9,8 @@
 </selection-tabs>
 
 <search-status-bar
-  ng-show="!vm.isLoading()"
+  class="search-status-bar"
```

Comment:
This class is applied to the top-level `<div>` rendered by the `<search-status-bar>` so it shouldn't be needed here as well. Note that when this template is converted, the Angular components will be replaced by a direct call to `SearchStatusBar` and the `<search-status-bar>` DOM element will disappear.